lib/plugins/aws/tests/naming.js

```diff
@@ -0,0 +1,435 @@
+'use strict';
+
+const expect = require('chai').expect;
+
+const SDK = require('../provider/awsProvider');
+const Serverless = require('../../../Serverless');
+
+describe('#naming()', () => {
+  let options;
+  let serverless;
+  let sdk;
+
+  beforeEach(() => {
+    options = {
+      stage: 'dev',
+      region: 'us-east-1',
+    };
+    serverless = new Serverless(options);
+    sdk = new SDK(serverless, options);
+  });
+
+  it('#normalizeName()', () => {
+    it('should capitalize the first letter', () => {
+      expect(sdk.naming.normalizeName('name')).to.equal('Name');
+    });
+    it('should have no effect on caps', () => {
```

Comment:
Maybe some space here so that all tests follow the same pattern. (This pattern might be found in other tests as well)